README.md

```diff
@@ -17,3 +17,10 @@ This repository contains the following implementations of the RGF algorithm:
 - [RGF](https://github.com/RGF-team/rgf/tree/master/RGF): original implementation from the paper;
 - [FastRGF](https://github.com/RGF-team/rgf/tree/master/FastRGF): multi-core implementation with some simplifications;
 - [rgf_python](https://github.com/RGF-team/rgf/tree/master/python-package): wrapper of both RGF and FastRGF implementations for Python.
+
+## Documentation
```

Comment:
To be honest, I'm against that in general README links to implementation-specific docs are placed. They can be found just after clicking to the implementation title in the list above.

I think the link to the `AWESOME_RGF` doc can be introduces this way:

>You may want to get interesting information about RGF from the posts collected in [Awesome RGF](link here).